versions/3.0.md

```diff
@@ -1464,7 +1464,7 @@ When passing complex objects in the `x-www-form-urlencoded` content type, the de
 
 ##### Special Considerations for `multipart` content
 
-It is common to use `multipart/form-data` as a `Content-Type` when transferring request bodies to operations.  In contrast to 2.0, a `schema` is required to define the input parameters to the operation when using `multipart` content.  This allows complex structures as well as supports mechanisms for multiple file uploads.
+It is common to use `multipart/form-data` as a `Content-Type` when transferring request bodies to operations.  In contrast to 2.0, a `schema` is required to define the input parameters to the operation when using `multipart` content.  This allows complex structures as well as supporting mechanisms for multiple file uploads.
```

Comment:
The original intent is not clear:
a) "This supports mechanisms for multiple file uploads", or
b) "This allows [complex structures and/or supporting mechanisms] ..." where "supporting mechanism" is a noun phrase.

It could be simpler and more concise instead: "This allows for multiple file uploads." as it, it begs the questions "what complex structures? What supporting mechanisms?" Need to find out what the original intent was.